### PR TITLE
Add gclib

### PR DESCRIPTION
**Pull request by davebx**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/gclib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package:
+  name: gclib
+  version: "907a9d"
+
+build:
+    number: 0
+    skip: False
+
+source:
+  git_url: https://github.com/gpertea/gclib.git
```
Comment (bgruening) [inline recipes/gclib/meta.yaml]: On github you can use this URL: `https://github.com/gpertea/gclib/archive/907a9d6484b92ea0c357baf2da5d85bc6e6b82b1.tar.gz`
+ a checksum and everything looks great!

File: recipes/gclib/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package:
+  name: gclib
+  version: "907a9d"
```
Comment (bgruening) [inline recipes/gclib/meta.yaml]: I'm not sure Conda can handle this in a long run. Is the next hash smaller or greater than this one?
Can you use 0.0.0.1 or something.

File: recipes/gclib/meta.yaml (hunk 1 above)
Comment (davebx) [inline recipes/gclib/meta.yaml]: This is the syntax they had in https://conda.io/docs/building/meta-yaml.html#source-from-git

Comment (bgruening) [inline recipes/gclib/meta.yaml]: @davebx this is correct. At bioconda we do backup all tarballs on Galaxy depot, so we have a mirror of everything. This backup is much easier to do with real tarballs and not git urls. Hope this makes sense.

**Comment by bgruening** [comment]
@davebx please see here for more information: https://bioconda.github.io/guidelines.html#c-c

**Pull request merged**